spec/integration/root_login_spec.rb

```diff
@@ -10,10 +10,11 @@
 describe 'Root login' do
   include IntegrationHelpers::DefaultHelper
 
-  it 'lets root login via local ip' do
+  xit 'lets root login via local ip' do
```

Comment:
xit ?

Reply:
Right, since this spec is depending on the start page we skip it here, and reenable it inside when implementing the start page :)